yarn-project/end-to-end/src/e2e_event_logs.test.ts

```diff
@@ -38,63 +29,7 @@ describe('Logs', () => {
   afterAll(() => teardown());
 
   describe('functionality around emitting an encrypted log', () => {
-    it('emits multiple events as encrypted logs and decodes them one manually', async () => {
```

Comment:
Nuked this test because L1EventPayload.decryptAsIncoming no longer works as now the encryption is different.

Will nuke L1EventPayload and the TS decryption functionality in a followup PR to reduce the diff size here.